src/backend/controllers/resources-controller.js

```diff
@@ -59,12 +59,12 @@ class ResourcesController extends BaseController {
     this.findResources(request.params)
     const { recordId, actionId } = request.params
     view.record = await view.currentResource.findOne(recordId)
-    const { h, record } = view
+    const { record } = view
     view.customAction = {
       name: actionId,
       content: await view.currentResource
         .decorate()
-        .getRecordActions(h, record)[actionId]
```

Comment:
decorator already has helpers now, so we dont have to pass it from controller and view